lib/src/main/java/com/auth0/jwt/JWTCreator.java

```diff
@@ -312,7 +316,9 @@ public Builder withClaim(String name, Date value) throws IllegalArgumentExceptio
          * @param items the Claim's value.
          * @return this same Builder instance.
          * @throws IllegalArgumentException if the name is null.
+         * @deprecated Use {@linkplain #withClaim(String, List)} instead.
          */
+        @Deprecated
         public Builder withArrayClaim(String name, String[] items) throws IllegalArgumentException {
```

Comment:
Do we provide an alternate way on **verification** for asserting lists claims? See the not deprecated method below
     
```
Verification withArrayClaim(String name, String... items) throws IllegalArgumentException;
```

Reply:
We do not. It's probably something we should re-evaluate though. And if we do add those (and later deprecate these), we should do that in a separate PR.